src/components/Tutorial.vue

```diff
@@ -50,11 +51,11 @@ export default {
   components: {
     LessonLink
   },
-  data: self => {
+  data () {
```

Comment:
Could you help me understand why this instance of `data` doesn't need the `self` but others do?